recipes/r-dfdr/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '0.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-dfdr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/dfdr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/dfdr/dfdr_{{ version }}.tar.gz
+  sha256: cabbe7b9c5760945defbd7716a71b7667e3b01237ef5f77c860055437508f1e8
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r6
+    - r-pryr
+    - r-purrr
+    - r-rlang
+  run:
+    - r-base
+    - r-r6
+    - r-pryr
+    - r-purrr
+    - r-rlang
+
+test:
+  commands:
+    - $R -e "library('dfdr')"           # [not win]
+    - "\"%R%\" -e \"library('dfdr')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=dfdr
+  license: GPL-2.0-only
+  summary: Implementation of automatically computing derivatives of functions (see Mailund Thomas
+    (2017) <doi:10.1007/978-1-4842-2881-4>). Moreover, calculating gradients, Hessian
+    and Jacobian matrices is possible.
+  license_family: GPL2
```

according to the cran meta data below:
```suggestion
  license: GPL-3.0-only
  summary: Implementation of automatically computing derivatives of functions (see Mailund Thomas
    (2017) <doi:10.1007/978-1-4842-2881-4>). Moreover, calculating gradients, Hessian
    and Jacobian matrices is possible.
  license_family: GPL3
```